crates/server/src/datastore/postgres/models.rs

```diff
@@ -91,6 +91,7 @@ pub struct NewCheckpoint<'a> {
 
 #[derive(Queryable)]
 #[diesel(table_name = checkpoints)]
+#[allow(dead_code)]
```

Comment:
What was marked dead here? I'm surprised anything in a `#[derive(Queryable)]` would be...

Reply:
The timestamp fields. The query fills them in, but nothing reads them.

Reply:
Ah yeah I guess `Queryable` would be read-only wouldn't it? :+1: